deis/azuredeploy.json

```diff
@@ -0,0 +1,205 @@
+{
```

Comment:
Can you define endpoints on VMs? It seems all ports are open by default. I haven't looked how to define load balancer in template yet, but yes ideally we need to add a load balancer.